src/Api/Controllers/DevicesController.cs

```diff
@@ -231,4 +235,22 @@ public async Task<bool> GetByIdentifier(string email, string identifier)
         var device = await _deviceRepository.GetByIdentifierAsync(identifier, user.Id);
         return device != null;
     }
+
+    [RequireFeature(FeatureFlagKeys.DeviceTrustLogging)]
+    [HttpPost("lost-trust")]
+    public async Task PostLostTrust()
+    {
+        var user = await _userService.GetUserByPrincipalAsync(User);
```

Comment:
⚠️ shouldn't we be able to use the user off the `CurrentContext` here without having to re-hit the db?

Reply:
Done!